### ISSUE TITLE
Docker for Mac - Error Code 404

### ISSUE THREAD

**Issue by lexicalunit** [dldl/sphinx-preview#4]
At first I kept getting the error that it couldn't connect to `localhost:2375`. After some searching I found out that Docker for Mac [doesn't expose that port](https://github.com/docker/for-mac/issues/770) directly. The workaround is to use `socat`, like so:

```shell
run -d -v /var/run/docker.sock:/var/run/docker.sock -p 127.0.0.1:2375:1234 bobrik/socat TCP-LISTEN:1234,fork UNIX-CONNECT:/var/run/docker.sock
```

That got me a little further, but I also had to manually pull the sphinx server image:

```shell
docker pull dldl/sphinx-server:latest
```

Now I'm stuck with this error:

<img width="682" alt="screen shot 2017-04-25 at 6 06 03 pm" src="https://cloud.githubusercontent.com/assets/1903876/25411363/e8df319e-29e1-11e7-9c59-76bafa5b6638.png">

This is what I get when I run a preview for an empty rst file. Thoughts?

**Comment by quentinus95** [comment]
Hi, thanks for reporting the issue, I'll have a Mac next week to be able to reproduce and fix the issue.

I'll keep you informed.

**Comment by quentinus95** [comment]
To be fixed first:

- bad condition on js file (currently using Windows configuration on MacOS...)
- bad file urls

**Comment by quentinus95** [comment]
The issue may be fixed thanks to 0.1.1 release.

Please see my comment on issue #5.
